lite/examples/sound_classification/android_compose/app/src/main/java/org/tensorflow/lite/examples/soundclassifier/compose/ui/SoundClassifierScene.kt

```diff
@@ -0,0 +1,185 @@
+          onIntervalChanged = viewModel::setClassificationInterval,
+          modifier = Modifier.padding(innerPadding),
+        )
+      }
+    }
+  }
+
+}
+
+@Composable
+private fun SoundClassifierBody(
+  probabilities: List<Category>,
+  classifierEnabled: Boolean,
+  interval: Long,
+  onClassifierToggle: (Boolean) -> Unit,
+  onIntervalChanged: (Long) -> Unit,
+  modifier: Modifier = Modifier,
+) {
+  Column(
+    modifier = modifier.padding(16.dp)
+  ) {
+    ControlPanel(
+      inputEnabled = classifierEnabled,
+      interval = interval,
+      onInputChanged = onClassifierToggle,
+      onIntervalChanged = onIntervalChanged,
+    )
+
+    Divider(modifier = Modifier.padding(vertical = 24.dp))
+
+    if (classifierEnabled) {
+      probabilities.let { itemList ->
+        LazyColumn {
+          itemsIndexed(
+            items = itemList,
+            key = { _, item -> item.label }
+          ) { index, item ->
+            ProbabilityItem(text = item.label, progress = item.score, index = index)
+          }
+        }
+      }
+    }
+  }
+}
+
+@Composable
+fun ControlPanel(
+  inputEnabled: Boolean,
+  interval: Long,
+  onInputChanged: ((Boolean) -> Unit) = {},
+  onIntervalChanged: (Long) -> Unit = {},
+) {
+  Row {
+    val labelText = stringResource(id = R.string.label_input)
+    Text(labelText, style = MaterialTheme.typography.body1)
+    Switch(
+      checked = inputEnabled,
+      onCheckedChange = onInputChanged,
+      modifier = Modifier.padding(start = 16.dp),
+      colors = SwitchDefaults.colors(checkedThumbColor = orange500)
+    )
+  }
+  Spacer(modifier = Modifier.height(12.dp))
+  Row(verticalAlignment = Alignment.CenterVertically) {
+    val labelText = stringResource(id = R.string.label_classification_interval)
+    Text(labelText, style = MaterialTheme.typography.body1)
+    Slider(
+      enabled = inputEnabled,
+      value = interval / 1000f,
+      onValueChange = { onIntervalChanged((it * 1000L).toLong()) },
+      modifier = Modifier
+        .padding(start = 8.dp)
+        .fillMaxWidth(),
+      colors = SliderDefaults.colors(
+        thumbColor = gray800,
+        activeTrackColor = gray800,
+      )
+    )
+  }
+}
+
+@Composable
+fun ProbabilityItem(text: String, progress: Float, index: Int = 0) {
+  val indicatorColor = progressColorPairs[index % 3].second
+  val backgroundColor = progressColorPairs[index % 3].first
+
+  Row(
+    verticalAlignment = Alignment.CenterVertically
+  ) {
+    Text(
+      text,
+      modifier = Modifier.width(92.dp),
+      style = MaterialTheme.typography.body2,
+    )
+    LinearProgressIndicator(
+      progress = progress,
+      modifier = Modifier
+        .height(52.dp)
+        .padding(start = 12.dp, top = 16.dp, end = 16.dp, bottom = 16.dp)
+        .clip(MaterialTheme.shapes.medium)
+        .fillMaxWidth(),
+      color = indicatorColor,
+      backgroundColor = backgroundColor
+    )
+  }
+}
+
+val DummyCategories = listOf(
```

Comment:
Could you rename it to a less sensitive name? e.g. `SampleCategories`

Reply:
Thanks! Will do that.